File: Web.Admin/ClientApp/src/app/models/dependency-lite.ts
export class DependencyLite {

  id: number;
  name: string;
  type: string;
  description: string

  public DependencyLite(id: number,
    name: string,
    type: string,
    description: string) {
    this.id = id;
    this.name = name;
    this.type = type;
    this.description = description;
  };

}

File: Libraries/Internal/NIST.CVP.Libraries.Internal.ACVPCore/Models/DependencyLite.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace NIST.CVP.Libraries.Internal.ACVPCore.Models
{
	public class DependencyLite
	{
		public long ID { get; set; }
		public string Name { get; set; }
		public string Type { get; set; }
		public string Description { get; set; }
	}
}
